### PR TITLE
[WEB-5732] Add feedback form to NPS page

### PR DESCRIPTION
**Pull request by luke-wyeth**
this adds the new SurveyAdd to accommodate the new endpoint created on the Profile Service as part of this story and [WEB-5868] SWS Profile Service - `POST /survey` endpoint (https://serato.atlassian.net/browse/WEB-5868)

[WEB-5868]: https://serato.atlassian.net/browse/WEB-5868

### CODE REVIEW

File: src/Profile/Command/SurveyAdd.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+
+namespace Serato\SwsSdk\Profile\Command;
+
+use Serato\SwsSdk\CommandBasicAuth;
+
+/**
+ * Add a User in the Beta program.
```
Comment (electric) [inline src/Profile/Command/SurveyAdd.php:9]: One small copy/paste remnant here

File: src/Profile/Command/SurveyAdd.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ */
+class SurveyAdd extends CommandBasicAuth
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getBody()
+    {
+        $args = $this->commandArgs;
+        unset($args['user_id']);
```
Comment (electric) [inline src/Profile/Command/SurveyAdd.php:28]: Maybe a small misunderstanding on this part (fair enough, it's not super obvious)

The reason we unset this here [in the example class](https://github.com/serato/sws-php-sdk/blob/master/src/Profile/Command/UserAddBetaProgram.php#L26) is to prevent the ID being encoded as part of the body since we include it [as part of the URL](https://github.com/serato/sws-php-sdk/blob/master/src/Profile/Command/UserAddBetaProgram.php#L43)

Therefore I think we shouldn't be unsetting this value here, as we still need to retrieve the user ID [when making the request](https://github.com/serato/sws-php-sdk/blob/master/src/Command.php#L120)

TLDR; I think you can remove this line, or simplify to:

```
    public function getBody()
    {
           return $this->arrayToFormUrlEncodedBody($this->commandArgs);
    }

**Review by electric — CHANGES_REQUESTED** [review]
A couple small pieces of feedback, but this is a great start - nice work @luke-wyeth

File: src/Profile/Command/SurveyAdd.php (hunk 2 above)
Comment (luke-wyeth) [reply src/Profile/Command/SurveyAdd.php]: oops, thanks! I had removed it but accidentally undid my changes when I removed my debugging code 😢

**Review by electric — APPROVED** [review]
Looks good 😄
